### DISCUSSION TITLE
Provide good support for .NET Aspire with MAUI

### DISCUSSION THREAD

**Discussion by BretJohnson** [dotnet/maui#21064] [Ideas]
Let's use this discussion topic for what a good, productized experience for .NET Aspire with MAUI should be.

It's a companion item the experiment here: https://github.com/BretJohnson/aspire-mobile. The intent of the experiment is to get something functional, that folks can use today, in order to get feedback on how to improve the experience, through changes in MAUI itself, Visual Studio, Aspire, etc. This is the spot to capture that feedback.

Potential discussion topics:
- Launch experience - for the experiment, the MAUI app and AppHost are launched separately in Visual Studio. How does that feel in practice, workflow wise? How would you like it to work in an ideal world?
- Template(s) - what would you like to see in a MAUI + Aspire project template, that may be different than the project template in the experiment?
- Any MAUI SDK core changes that could make for a better experience, beyond what's in the experiment
- Dashboard - what else would you like to see there, for MAUI apps?
- Anything else that you'd like to see

**Comment by BartNetJS** [comment]
@BretJohnson, I'm experimenting with the .NET Aspire with MAUI.

Currently, I have to start my MAUI app (on Windows) in a second debug session. As there is no option like for the web based apps, to launch:
![image](https://github.com/dotnet/maui/assets/4865355/9bebf951-17f7-4d55-95ff-8944642aa78d)
Will it be an option to deploy for example to a android device from the portal?

I also tried 'azd up' to deploy the solution. What I noticed is that the MAUI app is also deployed, is that how you see it?
Will it be part of the azd up to get it in the App store (apple, google, microsoft))

Second, I've also added a hybrid blazor MAUI app. In the image above seen as 'mauiblazor'.
How can i exclude it to be deployed on 'azd up'?
